ui/appui-react/src/appui-react/childwindow/InternalChildWindowManager.tsx

```diff
@@ -257,6 +271,25 @@ export class InternalChildWindowManager implements FrameworkChildWindows {
     return outLocation;
   }
 
+  // This is needed due to the differences in how browsers calculate the
+  // inner vs outer width of windows. Calculations are different between
+  // browsers. We are starting with some of the mainly used browsers but can
+  // add more if issues arise with others.
+  private findBrowser(agent: string): string {
+    switch (true) {
+      case agent.indexOf("edge") > -1:
+        return "edge";
+      case agent.indexOf("firefox") > -1:
+        return "firefox";
+      case agent.indexOf("edg/") > -1:
+        return "chromium based edge";
```

Comment:
Since we use this for only 1 particular purpose and the only value that we appear to care is this one, should we reduce this to `const shouldUseOuterSize = agent.indexOf("edg/") > -1;` ? Seems that we are reducing the agent string, but with no clear reason at this point. Or do you foresee that we migth need completely different solutions depending on these browsers ?

Reply:
I think for now we can remove this part, we'll still have the PR history to get back with this solution when we need it, if it happens that this is the solution we need then.

Reply:
Removed